src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const employees = Array.from(document.querySelectorAll('li'));
+
+function getNumberFromString(str) {
+  return parseInt(str.dataset.salary.split(',').join('').split('$').join(''));
```

Comment:
u can make it easier
```suggestion
  return parseInt(str.dataset.salary.replace(',', '').slice(1));
```